### PR TITLE
ROC-3268 Update tests for enabling CCD

### PR DESCRIPTION
**Pull request by timja-kainos**

### CODE REVIEW

File: src/integration-test/helpers/clients/idamClient.ts [hunk 1]
```diff
@@ -31,13 +31,64 @@ export class IdamClient {
...
+    return request.get({
+      uri: `${baseURL}/testing-support/accounts/pin/${letterHolderId}`,
+      resolveWithFullResponse: true,
+      rejectUnauthorized: false,
+      json: false
+    })
+  }
+
+  /**
+   * Authorizes user
```
Comment (rlewan) [inline src/integration-test/helpers/clients/idamClient.ts]: That comment probably wasn't updated.

File: src/integration-test/helpers/clients/idamClient.ts [hunk 2]
```diff
@@ -31,13 +31,64 @@ export class IdamClient {
...
+   *
+   * @param {string} email
+   * @param upliftToken the pin user's authorization header
+   * @returns {Promise<string>}
+   */
+  static async upliftUser (email: string, upliftToken: string): Promise<string> {
```
Comment (rlewan) [inline src/integration-test/helpers/clients/idamClient.ts]: Would you mind renaming `upliftToken` to `userAuthenticationToken` or just `authenticationToken` like it's done in other places? This is just a normal user auth token and the parameter name makes it sound like something different.

Comment (timja-kainos) [reply src/integration-test/helpers/clients/idamClient.ts]: its not really though, its a specific token for a pin user, and the token in this context is only used for uplifting

**Comment by timja-kainos** [comment]
image

File: src/integration-test/helpers/clients/idamClient.ts (hunk 2 above)
Comment (pragnesh-patel) [reply src/integration-test/helpers/clients/idamClient.ts]: pinAuthenticationToken / pinUpliftToken ?

**Pull request merged**